README.md

```diff
@@ -1,56 +1,78 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Budget-app
 
-> Description the project.
+>  mobile web application where you can manage your budget:
+   you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+<a href="https://guides.rubyonrails.org/index.html" target="_blank" rel="noreferrer"> <img
+      src="https://user-images.githubusercontent.com/57408419/178755770-7704dd7c-7bf2-4225-ad65-be3f1e4d1e2d.png"
+      alt="react" width="120" height="70" /> </a>
+<a href="https://www.postgresql.org/" target="_blank" rel="noreferrer"> <img
+      src="https://user-images.githubusercontent.com/57408419/178756727-8bfad119-18c2-49eb-98be-8b2d4bd71dd6.jpeg"
+      alt="react" width="120" height="70" /> </a>
 
-## Live Demo (if available)
+- Ruby on Rails.
+- PostgreSQL.
 
-[Live Demo Link](https://livedemo.com)
+![badgetApp](https://user-images.githubusercontent.com/57408419/184454848-f5fa90bc-58ab-46ae-8e5a-29fed6fa8572.png)
 
+### [live badgetApp](https://damp-garden-36904.herokuapp.com/)
+### [live video](https://www.loom.com/share/ed72adfcaedc484cb391d77a33739d07https://www.loom.com/share/ed72adfcaedc484cb391d77a33739d07)
```

Comment:
- [ ] I can't able to view the video. Please access this video to the public so that reviewers can see your presentation.